Composer/packages/server/src/models/asset/assetManager.ts

```diff
@@ -272,36 +272,43 @@ export class AssetManager {
       const feedType = this.getFeedType();
 
       if (feedType === 'npm') {
-        return data.objects.map((result) => {
-          const { name, version, keywords, description = '' } = result.package;
-          const displayName = this.getPackageDisplayName(name);
-          const templateToReturn = {
-            id: name,
-            name: displayName,
-            description: description,
-            package: {
-              packageName: name,
-              packageSource: 'npm',
-              packageVersion: version,
-            },
-          } as BotTemplate;
-          if (isArray(keywords)) {
-            if (keywords.includes('bf-dotnet-functions') || keywords.includes('bf-dotnet-webapp')) {
-              templateToReturn.dotnetSupport = {
-                functionsSupported: keywords.includes('bf-dotnet-functions'),
-                webAppSupported: keywords.includes('bf-dotnet-webapp'),
-              };
+        const result: BotTemplate[] = await Promise.all(
```

Comment:
Is the behavior here to bail out even if you are not able to get one of the feed contents? Shouldt we be able to continue processing just the valid feeds?

Reply:
In addition, we need error handling around these API calls.

Reply:
- It is currently set up to skip feeds which you cannot get content for. If a feed is empty or the call to the feed throws an error then the calling function (`getCustomFeedTemplates`) skips that feed and jumps to the next one for processing.
- I have added additional error handling